### PR TITLE
[Commerzbank]  Added a new file type that should not be parsed

### LINKED ISSUE #1225: [Commerzbank] Buy of apple not being parsed
## Document
<details><summary>Apple 2.pdf</summary><p>

```json
[
  [
    "Umsatzdetails",
    "Max Mustermann",
    "www.commerzbank.de",
    "Download am",
    "25. Januar 2021, 16:29 Uhr",
    "Seite",
    "1",
    "/",
    "1",
    "IBAN:",
    "DE86 5904 0000 0573 2953 00  |  BIC COBADEFFXXX",
    "Kontonummer:",
    "573295300 | BLZ 59040000",
    "Kontowährung:",
    "EUR",
    "Kontoinhaber:",
    "Max Mustermann",
    "Buchungstag:",
    "16.06.2020",
    "Buchungstext:",
    "WP-KAUF GESCH.TG.15.06.2020",
    "V. 15.06.2020 77/001858",
    "APPLE INC.",
    "WPKNR: 865985 ISIN: US0378331005",
    "NENNWERT: 2,000",
    "GESCH.-NR: 000148300715/001",
    "Wertstellung:",
    "17.06.2020",
    "Betrag:",
    "- 601,30 EUR"
  ]
]

```

</p></details>

---
Issue submitted with https://import.tresor.one

### PR DESCRIPTION
**Pull request by apentermann**
Hi,

I added the funktion **detectedButIgnoredDocument** to the Commerzbank file and added the first file type.

That fixes #1225

### CODE REVIEW

**Review by dredav — COMMENTED** [review]
Thanks for your first PR! Your change looks good but can you add also a test case which will ensures that the code will work also in the future.

**Comment by dredav** [comment]
You can find an example for status code 7 test for the trade republic broker [here](https://github.com/tresorone/tresor-import/blob/master/tests/brokers/traderepublic.test.js#L638-L646).

**Comment by dredav** [comment]
Can you also check if this PR also resolve #1230 and #852?

**Comment by apentermann** [comment]
- added the test case, I hope I did it right
- All documents from #1230 and #852 results now with a status 7. I think they can be closed after merging this.

**Review by dredav — APPROVED** [review]

**Comment by dredav** [comment]
> I think they can be closed after merging this.
I added both as 'linked issue' (on the right side on the PR page). Closing this PR will automatically close liked issues.

File: tests/brokers/__mocks__/commerzbank/ignored/umsatzdetails.json [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    "Download am",
+    "25. Januar 2021, 16:29 Uhr",
+    "Seite",
+    "1",
+    "/",
+    "1",
+    "IBAN:",
+    "DE86 5904 0000 0573 2953 00  |  BIC COBADEFFXXX",
+    "Kontonummer:",
+    "573295300 | BLZ 59040000",
```
Comment (dredav) [inline tests/brokers/__mocks__/commerzbank/ignored/umsatzdetails.json:15]: Could you replace the IBAN and Kontonummer with zeros?

**Review by dredav — CHANGES_REQUESTED** [review]
Found one more thing...

File: tests/brokers/__mocks__/commerzbank/ignored/umsatzdetails.json (hunk 1 above)
Comment (apentermann) [reply tests/brokers/__mocks__/commerzbank/ignored/umsatzdetails.json]: Sorry, I didn't know that this data had to be removed.

**Review by dredav — APPROVED** [review]

File: tests/brokers/__mocks__/commerzbank/ignored/umsatzdetails.json (hunk 1 above)
Comment (dredav) [reply tests/brokers/__mocks__/commerzbank/ignored/umsatzdetails.json]: hmm some users don't care about the data but some did. So it's better we remove the stuff before putting it forever in our git repo.

**Pull request merged**